lighthouse-core/lib/dependency-graph/simulator/connection-pool.js

```diff
@@ -82,15 +82,21 @@ module.exports = class ConnectionPool {
         throw new Error(`Could not find a connection for origin: ${origin}`);
       }
 
+      // Make sure each origin has 6 connections available
+      // https://cs.chromium.org/chromium/src/net/socket/client_socket_pool_manager.cc?type=cs&q="int+g_max_sockets_per_group"
+      while (connections.length < 6) connections.push(connections[0].clone());
+      while (connections.length > 6) connections.pop();
```

Comment:
how would we end up with >6?  our network records indicate we have >6 even though that should be impossible?

Reply:
If Chrome killed connections and needed to reopen them, we can kill this line though